ga4gh/cli.py

```diff
@@ -127,6 +127,31 @@ def printVariant(self, v):
         print()
 
 
+class ReferenceSetSearchRunner(object):
+    """
+    Runner class for the referencesets/search method.
+    """
+    def __init__(self, args):
+        rsr = protocol.GASearchReferenceSetsRequest()
+        if rsr.accessions:
+            rsr.accessions = args.accessions.split(",")
+        # TODO google-specific assemblyId not a valid field,
+        # referenceSetId (contrary to goog docs) not implemented
+        del rsr.__dict__['assemblyId']
```

Comment:
It looks like we'll need to put in some workarounds for Google in general here. However, we don't want to make our client Google specific. Perhaps we could add in a command line argument to turn on Google workarounds?